201-sql-auditing-server-policy-to-blob-storage/azuredeploy.json

```diff
@@ -0,0 +1,83 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+      "sqlServerName": {
+        "type": "string",
+        "defaultValue": "[concat('server-', uniqueString(resourceGroup().id, deployment().name))]",
+        "metadata": {
+          "description": "Name of the SQL server"
+        }
+      },
+    "location": {
+      "type": "string",
+      "defaultValue": "[resourceGroup().location]",
+      "metadata": {
+        "description": "Location for all resources."
+      }
+    },
+    "sqlAdministratorLogin": {
+      "type": "string",
+      "metadata": {
+        "description": "The administrator username of the SQL Server."
+      }
+    },
+    "sqlAdministratorLoginPassword": {
+      "type": "securestring",
+      "metadata": {
+        "description": "The administrator password of the SQL Server."
+      }
+    },
+    "storageAccountName": {
+      "type": "string",
+      "defaultValue": "[concat('sqlaudit',uniqueString(resourceGroup().id))]",
+      "metadata": {
+        "description": "The name of the auditing storage account."
+      }
+    }
+  },
+  "variables": {
+    
+  },
+  "resources": [{
```

Comment:
format the JSON (e.g. SHIFT+ALT+F in VS Code)

Reply:
done